src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/system_information.html.twig

```diff
@@ -27,261 +27,281 @@
 
 {% block content %}
 <div class="row">
-    <div class="col-lg-6">
-        <div class="card">
-            <h3 class="card-header">
-                <i class="material-icons">info_outline</i> {{ 'Configuration information'|trans }}
-            </h3>
-            <div class="card-block">
-                <div class="card-text">
-                    <p>{{ 'This information must be provided when you report an issue on our bug tracker or forum.'|trans }}</p>
-                </div>
-            </div>
+  <div class="col-lg-6">
+    <div class="card">
+      <h3 class="card-header">
+        <i class="material-icons">info_outline</i> {{ 'Configuration information'|trans }}
+      </h3>
+      <div class="card-block">
+        <div class="card-text">
+          <p>{{ 'This information must be provided when you report an issue on our bug tracker or forum.'|trans }}</p>
```

Comment:
If we change our bug tracker, no need to change the wording ;)

Reply:
Well, the 'bug tracker' used to refer to the Forge so it might be a little confusing for the users, especially now that the footer has been removed from the back office.

Reply:
Whatever you decide guys. You both made good points!

Reply:
You're right @LouiseBonnard. Ok for me for Github ;)

Reply:
Resolved in af9eade202539b38dd272839ce34570b813373d0